app/controllers/routes_controller.rb

```diff
@@ -1,13 +1,55 @@
 class RoutesController < ApplicationController
-  helper_method :routes
+  helper_method :routes, :route
 
   def index
     @model_attributes = model_attributes(Route)
   end
 
+  def new
+    @route = Route.new
+  end
+
+  def create
+    @route = Route.new(route_params)
+
+    if route.save
```

Comment:
```suggestion
    if @route.save
```
otherwise its calling the helper and that seems kinda confusing